rebvo:
  global:
    VideoNetEnabled: true
    VideoNetHost: 127.0.0.1
    VideoNetPort: 2708
    BlockingUDP: false
    SaveLog: true
    LogFile: rebvo_log.m
    TrayFile: rebvo_tray.txt

  Camera:
    ZfX: 386.9947285032803
    ZfY: 386.9020486508100
    PPx: 189.31522480114646
    PPy: 137.56218844645235
    KcR2: -2.917e-01
    KcR4: 8.228e-02
    KcR6: -0.001957
    KcP1: 5.333e-05
    KcP2: -1.578e-04
    ImageWidth: 640
    ImageHeight: 512
    FPS: 30
    UseUndistort: false
    Rotate180: false

  imu:
    ImuMode: 0

  transVlp2Cam:
    tx: 0.0326
    ty: -0.1328
    tz: -0.0049
    rx: 1.59436
    ry: -1.69065
    rz: 0.0176359

  Detector:
    Sigma0: 2.7818
    KSigma: 1.2599
    ReferencePoints: 12000
    MaxPoints: 40000
    TrackPoints: 10000
    DetectorThresh: 0.1
    DetectorAutoGain: 5e-7
    DetectorMaxThresh: 10
    DetectorMinThresh: 0.01
    DetectorPlaneFitSize: 2
    DetectorPosNegThresh: 0.4
    DetectorDoGThresh: 0.155259868922420

  TrackMaper:
    SearchRange: 40
    QCutOffNumBins: 100
    QCutOffQuantile: 0.9
    TrackerIterNum: 10
    TrackerInitType: 2
    TrackerInitIterNum: 2
    TrackerMatchThresh: 1
    MatchThreshModule: 1
    MatchThreshAngle: 45
    MatchNumThresh: 0
    ReweigthDistance: 2
    RegularizeThresh: 0.5
    LocationUncertaintyMatch: 2
    ReshapeQAbsolute: 1e-4
    ReshapeQRelative: 1e-2
    LocationUncertainty: 1
    DoReScaling: false
    GlobalMatchThreshold: 500

  ProcesorConfig:
    SetAffinity: true
    CamaraT1: 1
    CamaraT2: 2
    CamaraT3: 3

